compiler_opt/tools/generate_default_trace_test.py

```diff
@@ -111,9 +111,15 @@ def test_generate_trace(self, mock_get_runner):
         output_path=os.path.join(tmp_dir.full_path, 'output'),
         output_performance_path=os.path.join(tmp_dir.full_path,
                                              'output_performance'),
-    ):
+        keys_file=os.path.join(tmp_dir.full_path, 'keys_file')):
       generate_default_trace.generate_trace()
 
+    with open(
+        os.path.join(tmp_dir.full_path, 'keys_file'),
+        encoding='utf-8') as keys_file:
+      keys = [key_line.strip() for key_line in keys_file.readlines()]
+      self.assertListEqual(keys, ['default', 'default', 'default', 'default'])
```

Comment:
can we have a test that's not just saying "default"?

Reply:
I switched it up to add the pid and then just assert the start. Not sure it does that much though...

There's no easy way from what I can tell to pass a specific ID to each runner that we could then use to split the keys.